.policy.yml

```diff
@@ -0,0 +1,48 @@
+policy:
+  approval:
+  - or:
+    - Dependabot update
+    - Code change
+
+approval_rules:
+- name: Dependabot update
+  requires:
+    count: 1
+    teams:
+    - "coopnorge/engineering"
+  options:
+    invalidate_on_push: true
+    request_review:
+      enabled: true
+      mode: all-users
```

Comment:
It would be nice to set this to random users, but how do we get the policy bot to request more than 1 review?

Reply:
ah you want to request more people? ah that i've been wondering as well.

Reply:
looks the amount of users is set the count in the rule to https://github.com/palantir/policy-bot/blob/909e077c5058753b78a2f6dc8b1965488f30fac0/policy/reviewer/reviewer.go#L272

Reply:
we could make a feature request to add a percentage (or number) to the setting of random-users. Feedback might be that we need to use teams

Reply:
Should we set it to teams and see how it behaves?

Reply:
that will be similar do how it works with CODEOWNERS. So also very simular to 'all-users'. The 'all-users' is actually very similar to 'teams' but does not require the team to have push access to the repo (which all org users already have)

Reply:
aha, what do we prefer in that case, that the reviewers have concrete push access or not for a repo like `go-datadog-lib`?

Reply:
for the future i would like to not use the `teams` feature since that requires extra (and unuseful for us) config on the repository. So i prefer `all-useres` or `random`

Reply:
`all-users` it is.